### PR TITLE
Tests for utf8 imports

### PR DESCRIPTION
**Pull request by kristoferbaxter**
Cannot reproduce #240 based on latest master.

@filipesilva Can you tell if this tests what you're expecting to see?

### CODE REVIEW

**Comment by filipesilva** [comment]
@kristoferbaxter this test is what I expect to see, yes.

I think there might be some problem on more complex codebases though... I added a repro in https://github.com/ampproject/rollup-plugin-closure-compiler/issues/240#issuecomment-569676337.

File: test/import/fixtures/utf8-lazy.esm.default.js [hunk 1]
```diff
@@ -0,0 +1 @@
+import { c as ɵɵbar } from './chunk-5275c9cc.js';\u0275\u0275bar();
```
Comment (filipesilva) [inline test/import/fixtures/utf8-lazy.esm.default.js]: I find it a bit surprising that the import itself uses `ɵɵbar` but the function call uses `\u0275\u0275bar`. I understand it works, just find it odd.

Comment (filipesilva) [reply test/import/fixtures/utf8-lazy.esm.default.js]: I also wonder if the renaming back to `ɵɵbar` is necessary at all. The original declaration was renamed to `c` anyway.

**Comment by kristoferbaxter** [comment]
Closing in favor of #277.

**Pull request closed without merging**